CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.14.0 (2021-03-08)
+
+- Add `skyThemeIf` and `skyThemeClass` directives [#204](https://github.com/blackbaud/skyux-theme/pull/204)
```

Comment:
Missing closing period `.`. Also, should this be `Added` to be consistent with our past tense?